build.py

```diff
@@ -55,8 +58,10 @@
     # Step 4a - figure out which schemas are embedded and which are linked
     embedded = set()
     linked = set()
+    class_module_dict={}
```

Comment:
"dict" is a very generic name. I suggest renaming this variable `class_to_module_map`